### PR TITLE
display student faces on arrow:hover

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: source/javascripts/modal_student.js [hunk 1]
```diff
@@ -51,16 +51,32 @@ $(document).ready(function(){
 
   function initArrows(prev, next) {
     if(prev) {
+      var name = $("#" + prev).find('h2').text();
+      $('[data-toggle="tooltip"].is-prev').tooltip('enable')
       $('.modal-student-arrow.is-prev').attr('data-id', prev);
+      $('.modal-student-arrow.is-prev span').css('background', "linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('" + $("#" + prev).find('img').attr('src') + "')")
```
Comment (cveneziani) [inline source/javascripts/modal_student.js]: Extract the url in a variable

File: source/javascripts/modal_student.js [hunk 2]
```diff
@@ -51,16 +51,32 @@ $(document).ready(function(){
...
     }
 
     if(next) {
+      var name = $("#" + next).find('h2').text();
+      $('[data-toggle="tooltip"].is-next').tooltip('enable')
+      $('.modal-student-arrow.is-next').attr('data-original-title', name)
+      $('.modal-student-arrow.is-next').attr('title', name)
+      $('.tooltip-inner').text(name)
       $('.modal-student-arrow.is-next').attr('data-id', next);
+      $('.modal-student-arrow.is-next span').css('background', "linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('" + $("#" + next).find('img').attr('src') + "')")
```
Comment (cveneziani) [inline source/javascripts/modal_student.js]: Same same

File: source/layouts/_modal_student.html.erb [hunk 3]
```diff
@@ -2,10 +2,12 @@
   <div class='modal-student-cross'><i class='mdi mdi-close'></i></div>
   <div>
     <div class='modal-student-arrows'>
-      <a class='modal-student-arrow is-prev'>
+      <a class='modal-student-arrow is-prev' data-toggle="tooltip" data-container='body' data-trigger='hover' data-placement="right" title="Tooltip on right">
```
Comment (cveneziani) [inline source/layouts/_modal_student.html.erb]: Remove the useless title

File: source/layouts/_modal_student.html.erb [hunk 4]
```diff
@@ -2,10 +2,12 @@
...
+        <span class='modal-student-arrow-img'></span>
         <i class='mdi-chevron-left mdi'></i>
       </a>
-      <a class='modal-student-arrow is-next'>
+      <a class='modal-student-arrow is-next' data-toggle="tooltip" data-container='body' data-trigger='hover' data-placement="right" title="Tooltip on right">
```
Comment (cveneziani) [inline source/layouts/_modal_student.html.erb]: Same same

**Comment by cveneziani** [comment]
Nice!

Review done.

**Comment by cveneziani** [comment]
We need a hand cursor on the arrows.

**Comment by ghost** [comment]
@cveneziani the `cursor: pointer` is here, but not on the gif.
- now you have the current_student avatar
  image
- some tooltips (with `cursor: pointer`)

**Comment by cveneziani** [comment]
Nice! :heart_eyes_cat:

**Pull request merged**